### ISSUE TITLE
Class Selector Highlight Issue

### ISSUE THREAD

**Issue by dggsax** [MarioRicalde/SCSS.tmbundle#215]
This has probably already been reported, but there's some weird syntax highlighting happening. When I try to do the following:
```
.class {
    // some css stuff
    &-second {
        // some more css stuff
    }
 }
```
to get:
```
.class {
    // some css stuff
}
.class-second {
    // some css stuff
    // some more css stuff
}
```
the package will highlight the "-second" as if it's an error. Screenshot below for reference.
![image](https://user-images.githubusercontent.com/13513445/34744604-c98c30c6-f55b-11e7-9395-415e223d8dbe.png)

Thanks!

**Comment by nypinstripes** [comment]
@MarioRicalde @dggsax Yes, thank you. Seeing it also, after the latest update. This is valid SCSS syntax ```&-xyz```, right now having to switch back to Sass > SCSS syntax each time i open a file, super annoying the 'always' setting won't stick either I guess cause they share the same name.  I'm also now noticing it for SVG tag name selectors like ```use``` & ```stroke:```

**Comment by higimo** [comment]
@nypinstripes sorry. I can't switch, this highlight in SCSS right now. This will be solved?

ST 3

**Comment by joshmakar** [comment]
Same thing here.

![capture](https://user-images.githubusercontent.com/9956440/36849643-8c0a2aa6-1d32-11e8-91b0-2b3a6df1a0e2.JPG)
